docs/aerodromes/classc/Amberley.md

```diff
@@ -26,44 +26,87 @@ AMB ADC owns the Class C airspace **in the AMB CTR** within 10nm of the YAMB ARP
+    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
     **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
     **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
 
 | Departure | Initial Constraint | Route |
 | --------- | ----------| --------- |
-| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
-| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
-| Byron 7 | `F190` at BINUP | COWIE KIWEE ZANET LOTMA SEMAJ |
-
-!!! note
-    Charts for the Coded Departures may be in YAMB AD2 Supplements 6.2.25.11 available here: [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+| Northern 3 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Northern 4 | `F200` at ANKED | ANKED JEZZA BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 3 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Central 4 | `F200` at ANKED | ANKED JEZZA BOBED VIRGE |
+| Byron 1 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
+| Byron 2 | `F200` at TUGUN | TUGUN LAGOB ZANET LOTMA SEMAJ |
 
 !!! note
     Since the Coded Departures are not SIDs, AMB ACD needs to write the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route to include all relevant tracking points.
 
+!!! abstract "Reference"
+    Charts for the Coded Departures are available in: [YAMB FIHA AD2 Supp Section 6.2.25](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
 
 ### Runway Operations
 Runways 15/33 are the primary runways at YAMB. 
 
-## Charts
+## Helicopter Operations
+
+### Taxiway Alpha
+
+Taxiway Alpha has been divided into portions for helicopter operations as follows;
+
+| Location | Name |
+| ------ | ----------|
+| Between A2 and A4     | Alpha North  |
+| Between A2 and RWY 04/22     | Alpha Centre |
+| Between A1 and RWY 04/22     | Alpha South |
+| TWY Q and A Junction     | Holding Point Quebec |
+
+!!! phraseology 
+    **CHOP41**: "Amberley Delivery, CHOP41, for winching ops within Alpha Centre, request clearance"  
+    **AMB ACD**: "CHOP41, Amberley Delivery, cleared to operate within Alpha Centre, not above A001, report ops complete"  
+    **CHOP41**: "Cleared for winching ops within Alpha Centre, not above A001, report ops complete"  
+
+<figure markdown>
+![YAMB Helicopter Taxiway Alpha](img/yamb_heli_taxiwayalpha.png){ width="600" }
+  <figcaption>YAMB Helicopter Taxiway Alpha</figcaption>
+</figure>
+
+### Helicopter Circuits
+
+To facilitate helicopter circuits, two areas have been established.
+
+**Choppers East**:  
+Utilising the threshold RWY 22 as a HLS, remaining within 600 m of threshold RWY 22
+and east of a line parallel to RWY 15/33 intersecting threshold RWY 22  
+
+**Choppers West**  
+Utilising the threshold RWY 04 as a HLS, remaining within 600 m of threshold RWY 04
+and west of a line parallel to RWY 15/33 intersecting threshold RWY 04.
```

Comment:
Also what circuit altitude is flown by helicopters?

Reply:
According to the AD2 SUP and mentioned in the circuit section;

"b) C-27J, light aircraft and helicopters not above `A010`. "

Reply:
Cheers